tests/co19/co19-dart2js.status

```diff
@@ -1883,7 +1920,7 @@ LibTest/async/Timer/Timer.periodic_A01_t01: Skip # Times out. Please triage this
 LibTest/isolate/Isolate/spawn_A01_t05: Skip # Times out. Please triage this failure
 WebPlatformTest/shadow-dom/events/retargeting-focus-events/test-002_t01: Skip # Times out. Please triage this failure
 
-[ $compiler == dart2js && $runtime == chrome && $system != macos ]
+[ $compiler == dart2js && $runtime == chrome && $system != macos && $system != windows && $system != linux]
```

Comment:
What system does this apply to, if it doesn't apply to macos, windows, or linux?  If these are not failing anymore, remove the section.